.pylintrc

```diff
@@ -1,2 +1,3 @@
 [MASTER]
-ignore=yoti_python_sdk/tests/**,examples/**
\ No newline at end of file
+ignore=tests,examples,protobuf
```

Comment:
Would it be better to qualify these with `yoti_python_sdk/` first, just in case we create any other folders matching these names that we might want to format? (unlikely, i know)

Reply:
Pylint configuration is a bit janky in the fact that you can't specify a path for a directory you want to ignore.  Instead you have to specify a base name which is part of the module you are linting i.e. to ignore `yoti_python_sdk/tests` I would exclude `tests` and then run `pylint yoti_python_sdk` and it would know to exclude it.

Reply:
Ok, let's leave it as is then 👍